operator/src/main/java/oracle/kubernetes/operator/KubernetesConstants.java

```diff
@@ -29,9 +29,9 @@
 
   String CONTAINER_NAME = "weblogic-server";
 
-  String DOMAIN_CONFIG_MAP_NAME = "weblogic-domain-cm";
+  String SCRIPT_CONFIG_MAP_NAME = "weblogic-domain-cm";
   String DOMAIN_DEBUG_CONFIG_MAP_SUFFIX = "-weblogic-domain-debug-cm";
-  String INTROSPECTOR_CONFIG_MAP_NAME_SUFFIX = "-weblogic-domain-introspect-cm";
+  String DOMAIN_CONFIG_MAP_NAME_SUFFIX = "-weblogic-domain-introspect-cm";
```

Comment:
Just an observation:  extensive refactoring/renaming makes it harder to code review.  Would be easier if such changes were in a separate pull.

Reply:
That would be difficult. In general, I refactor in order to make it easier to make changes. I don't know what refactorings I am going to need until I need to make those changes, and making them often leads to new refactorings.

The best way to address this, IMO, is for people to refactor aggressively as they go - then big after-the-fact refactorings would be rare.